packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -954,6 +954,76 @@ describe('cluster', () => {
     });
   });
 
+  test('addRotationMultiUser() with options', () => {
```

Comment:
Can we write a better description of what this test is checking?

Reply:
We are testing here that those rotation options are correctly passed: `automaticallyAfter`, `excludeCharacters` and `vpcSubnets`.

Same as here:
https://github.com/aws/aws-cdk/blob/e0f863a4c56041860e14c75b9aa5a6d35860fae6/packages/%40aws-cdk/aws-rds/test/instance.test.ts#L800

Reply:
OK, can we write something like that? 🙂